test/unit/src/server/handlers/t_launchpad.js

```diff
@@ -0,0 +1,130 @@
+import expect from 'expect';
+import nock from 'nock';
+
+import { completeSnapAuthorization } from '../../../../../src/server/handlers/launchpad';
+import { conf } from '../../../../../src/server/helpers/config.js';
+
+describe('completeSnapAuthorization', () => {
+  const session = { 'token': 'secret' };
+
+  before(() => {
+    const overrides = conf.stores['test-overrides'];
```

Comment:
We seem to do the same test config overrides in couple of places already.

Not something to fix in this branch, but maybe it would be worth extracting it to some helper functions or maybe we could just make it a part of test setup somehow.